### PR TITLE
Adds Hydra backup systemd and restore script

### PR DESCRIPTION
**Pull request by peeech**

### CODE REVIEW

File: overlays/holo-nixpkgs/holo-hydra-restore/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    rev = "cdc69008ff895fcfebf992a80c49efccc5bb476b";
+    sha256 = "0ir0pahm73j6gcaz4df2h2gwyy0ik1xji83qpqrir18ishla224r";
+  };
+
+  buildInputs = [ openssl restic ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp holo-hydra-restore $out/bin/
+    chmod +x $out/bin/holo-hydra-restore
```
Comment (filalex77) [inline overlays/holo-nixpkgs/holo-hydra-restore/default.nix:18]: Nitpick, but essentially the same thing:

```suggestion
    install -Dm +x holo-hydra-restore $out/bin/holo-hydra-restore
```

Comment (peeech) [reply overlays/holo-nixpkgs/holo-hydra-restore/default.nix:18]: That was my intention but somewhat I forgot to use `install` ;-)

**Comment by filalex77** [comment]
Needs a rebase and a force push @peeech

**Comment by peeech** [comment]
> Needs a rebase and a force push @peeech

Can't force push to protected branch. Instead - merging hydra in :-)

**Pull request merged**